mmv1/third_party/terraform/go.mod.erb

```diff
@@ -28,18 +28,18 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	go4.org/netipx v0.0.0-20231129151722-fdeea329fbba
 	golang.org/x/exp v0.0.0-20240409090435-93d18d7e34b8
-	golang.org/x/net v0.25.0
-	golang.org/x/oauth2 v0.20.0
-	google.golang.org/api v0.180.0
```

Comment:
is this upgrade needed to support the field in GA?

Reply:
It is. The ConfidentialInstanceType field was added to the GA API very recently.